products/plugins/commercial/README.md

```diff
@@ -0,0 +1,15 @@
+# Commercial
+
+The Shopware 6 commercial feature-set comprises myriad features, the sum of which provide additional support for businesses which require extended functionality within the Shopware 6 ecosystem.
+
+## Plugin Structure
+
+The commercial plugin is structured as a group of nested [sub-bundles](concepts/extensions/plugins-concept.md#symfony-bundles):
+
+## Setup
+
+Installation of the commercial plugin does not require special installation guidance, the installation steps are simply as detailed in our [plugin base guide](guides/plugins/plugins/plugin-base-guide#install-your-plugin).
```

Comment:
Maybe redirect to the pages on various features (so currently only to "custom-pricing")